scripts/runmariadb

```diff
@@ -17,6 +17,7 @@ if [ ! -f "$MARIADB_CERT_FILE" ] && [ -f "$MARIADB_KEY_FILE" ] ; then
 fi
 
 ln -sf /proc/self/fd/1 /var/log/mariadb/mariadb.log
+rm -f /usr/bin/mysqld_safe_helper # Avoid this script being used which results in a permission denied error.
```

Comment:
This doesn't look like an appropriate fix. In any case, we need a much more detailed explanation here than "permission denied error".

Reply:
I think what we need is correct permissions ` /var/log/mariadb/`. You're simply removing the mysql executable.